accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -715,13 +745,13 @@ It's a compile-time error if:
     a pre-feature library and all `base` or `final` superdeclarations
     are in platform libraries.)_
 
-    More formally:
-    A declaration *C* in library *L* has a declared interface *P*,
-    and *P* has any superdeclaration *S*, from a library *K*,
-    which is marked `base` or `final` _(including *S* being *P* itself)_,
-    and neither:
-    * *K* and *L* is the same library, mor
-    * *K* is a platform library and *L* is a pre-feature library.
+    More formally, it's a compile-time error if:
+    A `class`, `mixin-class`, `mixin` or `enum` declaration in library *L* has
+    a declared interface *P*, and *P* has any superdeclaration *S*,
+    from a library *K*, which is marked `base` or `final`
```

Comment:
Also, since you specify behaviour for `final` declaration subtyping on line 683, maybe move these `final` examples (that aren't already in the `final` section) over to there and remove talking about both `base` and `final` in this section?

It's nice to read about each modifier one at a time.

Reply:
This rule might just be about `base`, because it's impossible to have a super-declaration marked `final` in another library without *also* having a super-declaration marked `base` in that other library, because that's the only possible way to have a direct super-declaration from that library with the `final` declaration as super-declaration.

(I think. It's always worrisome to define rules that depend on *other* rules to be complete. If we remove or tweak the other rule, then this rule might start allowing something we didn't intend.
So, be complete in every rule, as if it was stand-alone, or be minimal and non-redundant? I'm aiming for the former.)